database/insertData/laos/03_template_license_MMC_MMD.js

```diff
@@ -0,0 +1,1559 @@
+                        category: INFORMATION
+                        parameters: { title: "## Education", style: "basic" }
+                      }
+                      {
+                        code: "Q10EducationLevel"
+                        index: 170
+                        title: "Organisation Category"
+                        elementTypePluginCode: "radioChoice"
+                        category: QUESTION
+                        isEditable: false
+                        parameters: {
+                          label: "Education Level"
+                          layout: "inline"
+                          options: ["Secondary", "University"]
+                        }
+                        defaultValue: {
+                          operator: "graphQL",
+                          children: [
+                            "query getUser($id: Int!){user(id: $id) {education}}",
+                            "",
+                            [
+                              "id"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "currentUser.userId"
+                              ]
+                            },
+                            "user.education"
+                          ]
+                        }
+                      }
+                      {
+                        code: "Q11Secondary"
+                        index: 180
+                        title: "Secondary"
+                        elementTypePluginCode: "longText"
+                        category: QUESTION
+                        isEditable: false
+                        parameters: { label: "Please enter details for secondary"}
+                        visibilityCondition: {
+                          operator: "="
+                          children: [
+                            {
+                              operator: "objectProperties"
+                              children: ["responses.Q10EducationLevel.text"]
+                            }
+                            "Secondary"
+                          ]
+                        }
+                        defaultValue: {
+                          operator: "graphQL",
+                          children: [
+                            "query getUser($id: Int!){user(id: $id) {secondary}}",
+                            "",
+                            [
+                              "id"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "currentUser.userId"
+                              ]
+                            },
+                            "user.secondary"
+                          ]
+                        }
+                      }
```

Comment:
The reason this isn't showing up is you need to build the response object using the `buildObject` operator, like you did with the others. The stored value for "secondary" is a plain string, but the default response needs to be `{ text: <string> }`

Reply:
Although, now that I think about it,  I thought @andreievg had made it so plain strings would be automatically converted to "{text:...}" when evaluating, but that might have just been for shortText. Will need to look into this further (bring it up at meeting this afternoon)

Reply:
No, as discussed in the meeting. Will need to use the `buildObject`, thanks for looking into that for me : )